staging/src/k8s.io/apiserver/pkg/server/healthz/healthz_test.go

```diff
@@ -148,10 +148,32 @@ func TestCheckerNames(t *testing.T) {
 	for _, tc := range testCases {
 		result := checkerNames(tc.have...)
 		t.Run(tc.desc, func(t *testing.T) {
-			if reflect.DeepEqual(tc.want, result) {
+			if !reflect.DeepEqual(tc.want, result) {
 				t.Errorf("want %#v, got %#v", tc.want, result)
 			}
 		})
 	}
+}
 
+func TestFormatCheckerNames(t *testing.T) {
```

Comment:
I would just fix the existing test so add the quotes (and so that it actually checks correctly ;-) ).  This just looks like a debug helper function, so the tests are already a little disproportionate...